src/deluge/model/clip/audio_clip.cpp

```diff
@@ -1229,11 +1229,11 @@ bool AudioClip::currentlyScrollableAndZoomable() {
 }
 
 void AudioClip::clear(Action* action, ModelStackWithTimelineCounter* modelStack, bool clearAutomation,
-                      bool clearNotesAndMPE) {
-	Clip::clear(action, modelStack, clearAutomation, clearNotesAndMPE);
+                      bool clearSequenceAndMPE) {
+	Clip::clear(action, modelStack, clearAutomation, clearSequenceAndMPE);
```

Comment:
readability: maybe accept an enum whatToClear that could be {SEQUENCE_MPE, AUTOMATION, EVERYTHING} instead of the two bools so that it's immediately clear when looking at a function call